src/ImageSharp.Web/ImageSharpTagHelper.cs

```diff
@@ -0,0 +1,366 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Text.Encodings.Web;
+using Microsoft.AspNetCore.Mvc.Razor.TagHelpers;
+using Microsoft.AspNetCore.Mvc.Routing;
+using Microsoft.AspNetCore.Mvc.TagHelpers;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using Microsoft.Extensions.Options;
+using SixLabors.ImageSharp.Processing;
+using SixLabors.ImageSharp.Web.Commands;
+using SixLabors.ImageSharp.Web.Middleware;
+using SixLabors.ImageSharp.Web.Processors;
+
+namespace SixLabors.ImageSharp.Web
```

Comment:
I think this class should be renamed to `ImageTagHelper` and the namespace changed to:
```suggestion
namespace SixLabors.ImageSharp.Web.TagHelpers
```

It might not be a big deal now, but if you'd like to add additional tag helpers later (e.g. picture/source or srcset), removing `ImageSharp` from the current class will help in naming them and also makes conditionally adding them to your view a little bit easier and non-repetitive (e.g. `@addTagHelper SixLabors.ImageSharp.Web.TagHelpers.ImageTagHelper, SixLabors.ImageSharp.Web`).

Reply:
Pushed the suggestion by accident there. Strongly disagree. They write this stuff one time. I'm not reducing the discoverability of type.

Reply:
Great suggestion! I've implemented that now.